packages/test-runner-commands/src/sendKeysPlugin.ts

```diff
@@ -2,31 +2,59 @@ import { TestRunnerPlugin } from '@web/test-runner-core';
 import type { ChromeLauncher, puppeteerCore } from '@web/test-runner-chrome';
 import type { PlaywrightLauncher } from '@web/test-runner-playwright';
 
+type TypePayload = { type: string; };
+type PressPayload = { press: string; };
+type DownPayload = { down: string; };
+type UpPayload = { up: string; };
```

Comment:
Decided to keep the API simple here instead of adding something like a `modifier` prop. While it is a bit more verbose for the user to send multiple sendKeys commands to trigger a key combination, it matches the APIs that Playwright and Puppeteer expose, which allows to defer to their documentation for the most part. Plus it keeps the logic in this plugin slim.